### PR TITLE
added modified attacking plans

### PR DESCRIPTION
**Pull request by punam173**
Added a plan that allows to attack all units at once (Both Crusaders and Prophets)

### CODE REVIEW

File: bots/pilgrim.js [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
-    }
-}
-
-export default pilgrim;
+import { SPECS } from 'battlecode';
+import nav from './nav.js';
+
+const pilgrim = {};
+
+pilgrim.takeTurn = (self) => {
```
Comment (shalika2019) [inline bots/pilgrim.js:6]: It Looks fine

File: bots/crusader.js [hunk 2]
```diff
@@ -121,7 +124,8 @@ crusader.takeTurn = (self) => {
     // }
 
     //Attack when number is more than 5
-    if(nearbyCrusaders.length >= 5 || a === 1){
+    var size = nearbyCrusaders.length + nearbyProphets.length
+    if(size >= 10 || a === 1){
```
Comment (shalika2019) [inline bots/crusader.js:128]: Why did you chose size>=10

**Review by shalika2019 — COMMENTED** [review]
Everthing looks good to me.

File: bots/crusader.js (hunk 2 above)
Comment (punam173) [reply bots/crusader.js:128]: I just choose a random number, it can be more than 10

**Pull request merged**